tests/unit/test_plugin.py

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright 2023 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import datetime
+from csp_billing_adapter.adapter import get_plugin_manager
+from csp_billing_adapter_google import plugin
+from csp_billing_adapter.config import Config
+
+# Comment out the following for now, not currently used which
+# flake8 flags as error
+# import pytest
+# from unittest.mock import Mock, patch
+
+
+good_config_file = 'tests/data/config_good.yaml'
+bad_config_file = 'tests/data/config_bad.yaml'
```

Comment:
You have only provided a config_good.yaml data file, but not the _bad or _missing variants...

Reply:
Missing does not exist by design. I put the def in for bad really for future place for when there are actually tests.